deployer/pom.xml

```diff
@@ -41,4 +41,11 @@
         <module>rest</module>
     </modules>
 
+    <dependencies>
```

Comment:
No problem to move the `slf4j-api` here, but I have two concerns:
1. in this PR, I don't see the `slf4j-api` dependency removed from the parent pom
2. `org.slf4j.impl` is not provided by `slf4j-api` but by an implementation (log4j, simple, or whatever). So, if the problem is around `org.slf4j.impl` is an issue, I don't get why moving `slf4j-api` will be a way to fix that. Maybe it's due to the `slf4j-log4j12` dependency (with `test` scope)  ?

Reply:
For the remove it's done on the main cave parent.